CLAUDE.md

```
@@ -310,8 +310,8 @@ All HTTP requests go through the unified `WandererNotifier.Infrastructure.Http`
 Example usage:
 ```elixir
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)